### PR TITLE
Bump quotas for COMET InSAR Workshop 2021 23-24 Nov 2021

### PR DESCRIPTION
**Pull request by manics**
See https://github.com/jupyterhub/mybinder.org-deploy/issues/2061

### CODE REVIEW

File: mybinder/values.yaml [hunk 1]
```diff
@@ -88,13 +88,30 @@ binderhub:
...
+            quota: 500
 
     GitRepoProvider:
       banned_specs:
         - ^https%3A%2F%2Fbitbucket.org%2Fnikiubel%2Fnikiubel.bitbucket.io.git/.*
         - ^https%3A%2F%2Fjovian.ml%2Fapi%2Fgit%2F.*
         - ^https%3A%2F%2Fframagit.org%2FCecGhesq%2Flic_mdf_nsi_1.*
         - ^(git|https?)%3A%2F%2Fnotabug.org%2FulslcuRux3Y%2F.*
+      spec_config:
+        - pattern: ^comet_licsar/licsbas.*
```
Comment (manics) [inline mybinder/values.yaml:109]: I've assumed GitLab repo limits can be increased in the same way as for GitHub..... is this correct?

Comment (yuvipanda) [reply mybinder/values.yaml:109]: GitLab has its own provider, no? This should probably go under that.

Comment (manics) [reply mybinder/values.yaml:111]: You're absolutely right 🤦. Fixed!

**Review by yuvipanda — APPROVED** [review]
Thanks!

**Comment by espiritocz** [comment]
hi, yes, i updated our request (now with final times/links), please see:
https://github.com/jupyterhub/mybinder.org-deploy/issues/2360#issuecomment-1295336898

cheers!

**Pull request merged**